tests/testthat/test-get_vartypes.R

```diff
@@ -0,0 +1,5 @@
+data(proj)
+
+test_that("get_vartypes", {
+  expect_true(any(!is.na(get_vartypes(proj)$class)))
+})
```

Comment:
can you also test that at least one of each variable types are identified correctly?

Reply:
I added tests for each variable type. Hope it's fine as it got a little messy to prepare the data accordingly. I tried to set it up so that it will still work even when our test data might be changed.

Reply:
great, thanks!